tutorials/bert_customop.py

```diff
@@ -0,0 +1,69 @@
+import base64
+import onnx
+
+from onnxruntime_extensions import get_library_path, make_onnx_model, pnp
+from pathlib import Path
+from transformers import AutoTokenizer
+from transformers.onnx import export, FeaturesManager
+
+# get an onnx model by converting HuggingFace pretrained model
+model_name = "bert-base-cased"
+model_path = Path("onnx-model/bert-base-cased.onnx")
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+if not model_path.exists():
+    if not model_path.parent.exists():
+        model_path.parent.mkdir(parents=True, exist_ok=True)
+    model = FeaturesManager.get_model_from_feature("default", model_name)
+    model_kind, model_onnx_config = FeaturesManager.check_supported_model_or_raise(model, feature="default")
+    onnx_config = model_onnx_config(model.config)
+    export(tokenizer,
+           model=model,
+           config=onnx_config,
+           opset=12,
+           output=model_path)
+
+# Load the onnx model as encoded string
+content = open(str(model_path), "rb").read()
+onnx_model_b64 = base64.b64encode(base64.decodebytes(content))
+
+# Load the corresponding vocabulary
+vocab_filepath = "data/bert-base-cased.txt"
+vocabulary = open(vocab_filepath, 'r', encoding='utf-8').read()
+
+# Create a new node for the custom op
+customop_node = onnx.helper.make_node(
```

Comment:
Why do we use ONNX API, instead of pnp.export here?

Reply:
We are loading the model only as a base64 encoded data. We aren't really exporting the model here to disk. We are merely building it in memory.